<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>*********************************************************************
**
** Copyright (C) 1992-2006 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Assistant of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License version 2.0 as published by the Free Software Foundation
** and appearing in the file LICENSE.GPL included in the packaging of
** this file.  Please review the following information to ensure GNU
** General Public Licensing requirements will be met:
** http://www.trolltech.com/products/qt/opensource.html
**
** If you are unsure which license is appropriate for your use, please
** review the following information:
** http://www.trolltech.com/products/qt/licensing.html or contact the
** sales department at sales@trolltech.com.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>DatabaseExplorerUI</class>
 <widget class="QWidget" name="DatabaseExplorerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database List</string>
  </property>
  <property name="toolTip">
   <string>Database List</string>
  </property>
  <property name="statusTip">
   <string>Database List</string>
  </property>
  <property name="whatsThis">
   <string>Database List</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="databaseList">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">Database</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionOpenTableInNewTabPage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../sqlexplorer.qrc">
     <normaloff>:/resources/images/table.png</normaloff>:/resources/images/table.png</iconset>
   </property>
   <property name="text">
    <string>Open Table In New TabPage</string>
   </property>
   <property name="statusTip">
    <string>Open Table In New TabPage</string>
   </property>
   <property name="whatsThis">
    <string>Open Table In New TabPage</string>
   </property>
  </action>
  <action name="actionShowMetadata">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Metadata  </string>
   </property>
   <property name="statusTip">
    <string>Show Metadata  </string>
   </property>
   <property name="whatsThis">
    <string>Show Metadata  </string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../sqlexplorer.qrc">
     <normaloff>:/resources/images/refresh.png</normaloff>:/resources/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Refresh</string>
   </property>
   <property name="whatsThis">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionConnectDatabase">
   <property name="icon">
    <iconset resource="../sqlexplorer.qrc">
     <normaloff>:/resources/images/db_add.png</normaloff>:/resources/images/db_add.png</iconset>
   </property>
   <property name="text">
    <string>Connect Database</string>
   </property>
   <property name="toolTip">
    <string>Connect Database</string>
   </property>
  </action>
  <action name="actionDisconnectDatabase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../sqlexplorer.qrc">
     <normaloff>:/resources/images/db_remove.png</normaloff>:/resources/images/db_remove.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionShowRecordCount">
   <property name="text">
    <string>Show Record Count</string>
   </property>
   <property name="toolTip">
    <string>Show Record Count</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../sqlexplorer.qrc"/>
 </resources>
 <connections/>
</ui>
